### PR TITLE
[DOC] - Update glossary

### PR DESCRIPTION
**Pull request by TomDonoghue**
Related to a point in #10 - this checks and updates the glossary.

### CODE REVIEW

**Review by claire98han — APPROVED** [review]
Looks great! Adding the subcategories really helps.

File: doc/glossary.rst [hunk 1]
```diff
@@ -3,25 +3,38 @@ Glossary
...
+--------------
+
+    firing rate
+        How fast an individual unit is firing, typically measured in Hz, as spikes per second.
+
+    coefficient of variation
+        A measure of the variability of unit firing, reflecting the dispersion of the data around the mean.
+
+    fano factor
+        A measure of the variability of unit firing, as as variant of the coefficient of variation.
```
Comment (maestapereira) [inline doc/glossary.rst:37]: There's an extra "as".

**Review by maestapereira — APPROVED** [review]
The updates are overall great!

File: doc/glossary.rst (hunk 1 above)
Comment (TomDonoghue) [reply doc/glossary.rst]: Good catch! Fixed

**Pull request merged**